### DISCUSSION TITLE
Can somebody help me with this?

### DISCUSSION THREAD

**Discussion by Lucky7k** [YimMenu/YimMenu#3592] [General]
![Captura de tela 2024-08-16 001700](https://github.com/user-attachments/assets/f285a657-adc5-4afa-b11a-d221af9c4c30)

i just put the car in the garage with my friend and now it seems that the car is cloned outside but for me its stuck in place but he can drive it 

but for me im still stuck in the same space

**Comment by iwanttodo1t** [comment]
I think there is a misunderstanding in this detection and suggest that the developer remove it
